chain/network/src/test_utils.rs

```diff
@@ -307,88 +277,110 @@ impl MockPeerManagerAdapter {
     }
 }
 
-// Start PeerManagerActor, and RoutingTableActor together and returns pairs of addresses
-// for each of them.
 #[cfg(feature = "test_features")]
-pub fn make_peer_manager_routing_table_addr_pair(
-) -> (Addr<PeerManagerActor>, Addr<RoutingTableActor>) {
-    let seed = "test2";
-    let port = open_port();
-
-    let net_config = NetworkConfig::from_seed(seed, port);
-    let store = create_test_store();
-    let routing_table_addr =
-        start_routing_table_actor(net_config.public_key.clone().into(), store.clone());
-    let peer_manager_addr = make_peer_manager(
-        store,
-        net_config,
-        vec![("test1", open_port())],
-        10,
-        routing_table_addr.clone(),
-    )
-    .0
-    .start();
-    (peer_manager_addr, routing_table_addr)
-}
+pub(crate) mod test_features {
```

Comment:
:+1: this looks much better. In general, I'd say that conditional imports are a smell, and that we should try to minimize those!